### ISSUE TITLE
Bsflrs maps crashes game

### ISSUE THREAD

**Issue by robrich82** [bibendovsky/bstone#462]
I imported the maps provided from https://legacy.3drealms.com/stuff/bsflrs.zip into a copy of the `MAPHEAD.BS6` and `MAPTEMP.BS6` using HWE.

I used floor 0-4 in the first episode and the other two maps in the second.

When I try and load episode 1 I get this error

```
BStone v1.2.12

[CRITICAL] [MAIN] LoadLevel
[MAIN] BFG (PS) at (59, 26).
```

If I load them in the original DOS version it works fine.

Here is the link to the maps inserted into the `MAPHEAD.BS6` and `MAPTEMP.BS6` https://mega.nz/file/lKIkXQ5a#8TzqvhZrspLeIzc-LnP-qU6a45fsrXe6l1xZDWutgiA

**Comment by bibendovsky** [comment]
Does these beta binaries (Windows 7 or higher) resolve the issue?

[bstone_windows_x86_v1_2_13-wip-ab48190e501ba7f9617ae6200fac07c1f375ea02.zip](https://github.com/bibendovsky/bstone/files/13780931/bstone_windows_x86_v1_2_13-wip-ab48190e501ba7f9617ae6200fac07c1f375ea02.zip)
[bstone_windows_x64_v1_2_13-wip-ab48190e501ba7f9617ae6200fac07c1f375ea02.zip](https://github.com/bibendovsky/bstone/files/13780933/bstone_windows_x64_v1_2_13-wip-ab48190e501ba7f9617ae6200fac07c1f375ea02.zip)

**Comment by robrich82** [comment]
Using the x64 one, I can't load any mission now. Not even standard ones, it just goes to a black screen then the program crashes with no error displayed.

**Comment by bibendovsky** [comment]
Fix for v1.2.12:
[bstone_windows_x86_v1_2_12-wip-7e7a7ebdced88fd5a9cc03c582cf65d7520cb503.zip](https://github.com/bibendovsky/bstone/files/13789071/bstone_windows_x86_v1_2_12-wip-7e7a7ebdced88fd5a9cc03c582cf65d7520cb503.zip)
[bstone_windows_x64_v1_2_12-wip-7e7a7ebdced88fd5a9cc03c582cf65d7520cb503.zip](https://github.com/bibendovsky/bstone/files/13789072/bstone_windows_x64_v1_2_12-wip-7e7a7ebdced88fd5a9cc03c582cf65d7520cb503.zip)

**Comment by robrich82** [comment]
I can confirm that has fixed the issue, thanks!
